.github/workflows/test.yml

```diff
@@ -96,3 +96,20 @@ jobs:
           github-token: ${{ secrets.github_token }}
           version-file: 'test-file.toml'
           version-path: 'package.version'
+
+  multiple-files:
```

Comment:
Could you keep this inline with the other ones and call it `test-multiple-lines`

Reply:
you mean `test-multiple-files`, right? :)

Reply:
done f66f6a2